docs/components/RelatedReadList.js

```diff
@@ -72,9 +85,12 @@ function tagInfo(tagChar) {
       tagClass = "archetype-tag-explanation";
       tag = "explanation";
       break;
-    default:
+    case "r":
       tagClass = "archetype-tag-reference";
       tag = "reference";
+      break;
+    default:
+      return new Error("unrecognized tag class " + tagChar);
```

Comment:
having permissive defaults will cause you headaches down the road. use "r" when you mean "r".

Reply:
Ya, OK I agree. Will update.